ffi/diplomat/src/timezone.rs

```diff
@@ -6,21 +6,25 @@ use icu_timezone::CustomTimeZone;
 
 #[diplomat::bridge]
 pub mod ffi {
+    #[cfg(feature = "icu_timezone")]
     use crate::datetime::ffi::ICU4XIsoDateTime;
     use crate::errors::ffi::ICU4XError;
+    #[cfg(feature = "icu_timezone")]
     use crate::provider::ffi::ICU4XDataProvider;
     use alloc::boxed::Box;
     use core::fmt::Write;
     use core::str::{self};
     use icu_timezone::CustomTimeZone;
     use icu_timezone::GmtOffset;
+    #[cfg(feature = "icu_timezone")]
     use icu_timezone::MetazoneCalculator;
     use icu_timezone::ZoneVariant;
 
     #[diplomat::opaque]
     #[diplomat::rust_link(icu::timezone::CustomTimeZone, Struct)]
     pub struct ICU4XCustomTimeZone(pub CustomTimeZone);
 
+    #[cfg(feature = "icu_timezone")]
     #[diplomat::opaque]
     #[diplomat::rust_link(icu::timezone::MetazoneCalculator, Struct)]
     pub struct ICU4XMetazoneCalculator(pub MetazoneCalculator);
```

Comment:
Suggestion: move `ICU4XMetazoneCalculator` into its own module so you don't need as many `cfg`s. I think you still need a single one for `maybe_calculate_metazone` but that's better than the 7 you currently need in this file.